gel/_internal/_dlist.py

```diff
@@ -516,6 +518,15 @@ def __init__(
             __mode__=__mode__,
         )
 
+    def _check_value(self, value: Any) -> _MT_co:
+        cls = type(self)
+
+        t = cls.type
+        if isinstance(value, cls.type):
+            return value
+
+        return t.model_validate(value)
```

Comment:
Here you can defer to some dunder in the abstract model that, in turn, calls `model_validate` in its `_pydantic.GelModel` overload.

Reply:
I don't understand what you suggest

Reply:
ok done